tests/integ/modin/test_numpy.py

```diff
@@ -69,9 +69,25 @@ def test_logical_operators():
 
 
 @pytest.mark.parametrize(
-    "np_ufunc", [np.add, np.subtract, np.divide, np.multiply, np.true_divide]
+    "np_ufunc",
+    [
+        np.add,
```

Comment:
For many of those, do we have any test with data that has NULL or NaN inside, i believe they might have some difference there, most about the mapping to Snowpark Pandas, for example, are they mapped to snowpark pandas missing value or actual NaN.
Maybe we can add some of those test in a different pr